packages/connectivity/src/scp-cf/jwt.ts

```diff
@@ -49,6 +49,24 @@ export function retrieveJwt(req: IncomingMessage): string | undefined {
   }
 }
 
+/**
+ * Checks if the given JWT is from the XSUAA or from an alternative issuer based on the iss property and the uaa domain of the XSUAA.
+ * @param decodedUserJwt - JWT to be checked.
+ * @returns True if JWT is issued by XSUAA
+ * @internal
+ */
+export function isXsuaaToken(decodedUserJwt: JwtWithPayloadObject): boolean {
```

Comment:
Would it be better to share the potential doc of the logic in this function?
So can see potential updates later in newer versions.

Reply:
discussed.
The jku is a standard key for saving domain, so you can check xsuaa url.